data/human/human missions.txt

```diff
@@ -900,7 +900,7 @@ mission "WR Star 1"
 	passengers 3
 	cargo "sensors" 2
 	on offer
-		dialog `A group of <bunks> scientists approaches you and asks if you will be headed to the Rim any time soon. "We're doing research on Wolf-Rayet stars," they explain, "and we're hoping to find a captain who can do a fly-by of <waypoints>." Scientific research in the Deep is notoriously well-funded, so they will probably pay you quite well.`
+		dialog `A group of <bunks> scientists approaches you and asks if you will be headed to the Rim any time soon. "We're doing research on Wolf-Rayet stars," they explain, "and we're hoping to find a captain who can do a flyby of <waypoints>." Scientific research in the Deep is notoriously well-funded, so they will probably pay you quite well.`
```

Comment:
Not needed. https://www.oed.com/dictionary/fly-by_n?tab=factsheet#3900446

Reply:
Also, I don't think head-shake is grammatically correct. Head-shake is a verb; headshake is a noun.

Reply:
> Not needed. https://www.oed.com/dictionary/fly-by_n?tab=factsheet#3900446

That might be the British way (though the [Cambridge dictionary would disagree](https://dictionary.cambridge.org/dictionary/english/flyby)), Merriam-Webster [only lists flyby](https://www.merriam-webster.com/dictionary/flyby) and actually redirects you to that page if you are looking for fly-by.

> Also, I don't think head-shake is grammatically correct. Head-shake is a verb; headshake is a noun.

So are you arguing for or against that fix?

Reply:
> the [Cambridge dictionary would disagree](https://dictionary.cambridge.org/dictionary/english/flyby)

Here are some of the examples given:
>  These fly-bys provided the space probe with enough momentum to travel all the way out to the asteroid belt. 

>  Another lunar fly-by was performed later that month at a distance of 34,300 km to further improve the orbital

The Wikipedia article listed is the one on "Cassini-Huygens" which uses both fly-by and flyby. Hence, I approved the changes, but left a comment stating that one of them was unnecessary, as either spelling works.

2. I am arguing for the fix.